tests/contracts/wallet_2.scilla

```diff
@@ -21,16 +21,21 @@ let mk_transaction_added_event =
 
 (* Event for communicating the execution of a transaction *)
 let mk_transaction_executed_event =
+  fun (tc : Uint32) =>
   fun (recipient : ByStr20) =>
   fun (amount : Uint128) =>
   fun (tag : String) =>
-    { _eventname : "Transaction executed"; recipient : recipient; amount : amount; tag : tag }
+    { _eventname : "Transaction executed"; transactionId : tc; recipient : recipient; amount : amount; tag : tag }
 
 (* Event for communicating that a transaction was signed *)
 let mk_signed_transaction_event =
   fun (tc : Uint32) =>
-  fun (signedBy : ByStr20) =>
-    { _eventname : "Transaction signed"; transactionId : tc; signedBy : signedBy }
+    { _eventname : "Transaction signed"; transactionId : tc }
+
+(* Event for communicating that a signature was revoked *)
+let mk_signature_revoked_event =
+  fun (tc : Uint32) =>
+    { _eventname : "Transact revoked"; transactionId : tc }
```

Comment:
Minor issue. Can we change `Transact revoked` to `Signature revoked`?

Reply:
> Minor issue. Can we change `Transact revoked` to `Signature revoked`?

fixed now.

Reply:
Looks all good now @vaivaswatha.